cvcl/views/assignment.py

```diff
@@ -155,14 +155,20 @@ def post(self, request, **kwargs):
             if vmd.timezone:
                 data_dict['timezone'] = vmd.timezone
 
-            if vmd.timezone:
+            if vmd.hostname:
                 data_dict['hostname'] = vmd.hostname
 
             user_data = '#cloud-config\n' + yaml.dump(data_dict, default_flow_style=False)
 
             if vmd.powershell_script:
                 user_data = '#ps1_sysnative\n' + vmd.powershell_script
 
+            if vmd.script_per_boot:
```

Comment:
The code is literally there: https://cloudinit.readthedocs.io/en/latest/topics/format.html#helper-script-to-generate-mime-messages